v0/destinations/hs/transform.js

```diff
@@ -292,22 +299,31 @@ function batchEvents(destEvents) {
     let batchEventResponse = defaultBatchRequestConfig();
 
     chunk.forEach(ev => {
-      const { email, updatedProperties } = getEmailAndUpdatedProps(
-        ev.message.body.JSON.properties
-      );
-      ev.message.body.JSON.properties = updatedProperties;
-      identifyResponseList.push({
-        email,
-        properties: ev.message.body.JSON.properties
-      });
-      metadata.push(ev.metadata);
+      const { properties } = ev.message.body.JSON
+      if (properties && !(properties instanceof Array)) {
+        identifyResponseList.push({...ev.message.body.JSON})
+        batchEventResponse.batchedRequest.body.JSON = {
+          inputs: identifyResponseList
+        };
+        batchEventResponse.batchedRequest.endpoint = `${ev.message.endpoint}/batch/create`
```

Comment:
test cases are missing for custom objects batch endpoint

Reply:
In test cases for router I see only one batch can be tested and each batch will be of same source type. If I add one for rETL others needs to be changed. suggest on what to do here

Reply:
Let's make another router test file for rETL

Reply:
Adding different files for test cases